lisp/dank-comment.el

```diff
@@ -133,6 +134,17 @@ FILL-COLUMN is used for filling the comment body."
                        formatted-placeholder)
   formatted-placeholder)
 
+(defun dank-comment--ewoc-pp (post-or-comment)
+  "EWOC pretty-printer for POST-OR-COMMENT.
+Only one ewoc can be active in a buffer, so the comments EWOC
+needs to be able to handle different objects.
+Optional POST-PP must be the post pretty-printer function."
+  (cond ((dank-comment-p post-or-comment) (insert (concat (dank-comment-format-metadata post-or-comment) "\n"
```

Comment:
@john2x  FYI, you could use `cl-typecase` to make this even more concise.  :)

Reply:
Thanks!